convert_state.py

```diff
@@ -1,60 +1,65 @@
+"""This code converts written state names to the 2 letter abbreviations, or the reverse"""
+import json
 
-
-
-import json  # import json for json processing
 def main():
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+    """
+    reads in the data from state_data json file
+    """
+    with open('state_data.json','r') as file:
+        state_data = file.read()
+        state_abbr_json = json.loads(state_data)
 
-    print(data)
-    
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
+    # dictionary of state abbreviations keys and state name values
+    state_abbr_to_longform = state_abbr_json  
+    state_longform_to_abbr = {} # dictionary of state name keys and state abbreviation values
     # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
+    for statekey, statevalue in state_abbr_to_longform.items() :
```

Comment:
state_key and state_value for proper variable name convention

Reply:
very good catch, thanks